src/PostgREST/Error.hs

```diff
@@ -198,6 +199,9 @@ instance JSON.ToJSON ApiRequestError where
     (Just $ JSON.String $ T.decodeUtf8 ("Invalid preferences: " <> BS.intercalate ", " prefs))
     Nothing
 
+  toJSON AggregatesNotAllowed = toJsonPgrstError
+    ApiRequestErrorCode23 "Use of aggregate functions is not allowed" Nothing Nothing
```

Comment:
Hm, codecov is complaining about this being untested.

Reply:
yes, I think that's because I enable this setting for the aggregate tests. I can add a test showing it fails when it's disabled, but is there a way to alter a setting on a per-test basis? I could only find examples on a per-module basis (e.g. https://github.com/PostgREST/postgrest/pull/2925/files#diff-f4402037436f4c7b85bee4c8c3d4820e194d9877598aee600b1f29af35abf543R247)

Reply:
I suppose I could add in a test in another module to show that it fails?

Reply:
You can use the same module, there's this example for the `RollbackSpec`:

https://github.com/PostgREST/postgrest/blob/125f10a60f9928981224557464365a723f7fc232/test/spec/Main.hs#L248-L258

Reply:
thanks! added.